lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -46,6 +46,13 @@
                             default: true
                         });
                     }
+                    if ( fields[1].options.length > 1 ) {
+                        fields[1].options.unshift({
+                            name: '- ' + gettext('Choose one') + ' -',
+                            value: '',
+                            default: true
+                        });
+                    }
 
```

Comment:
These 2 blocks of code are almost identical. It would be better to move to a function that is called twice passing in the array and position. e.g. 
```
addDefaultOption(array, index) {
  if ( array[index].options.length > 1 ) {
    array[index].options.unshift({
      name: '- ' + gettext('Choose one') + ' -',
      value: '',
      default: true
    });
  }
}
```